_data/tasks.yml

```diff
@@ -23,4 +23,7 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
-
+- title: Stateful Set
+  section:
+  - title: Upgrade from PetSet to Stateful Sets
```

Comment:
Changed it to two words.  But keep is as plural to be consistent with other documents